contributing/contributing-code.md

```diff
@@ -123,17 +123,11 @@ For further detail on integrating third-party services (e.g., fonts, embeds, etc
 
 ## Contributor License Agreement
 
-AMP requires all contributors to either sign an individual Contributor License Agreement or be covered by a corporate Contributor License Agreement in order to protect contributors and users in issues of intellectual property.
+AMP requires all contributors to agree to the OpenJSF Contributor License Agreement in order to protect contributors and users in issues of intellectual property.
 
-We recommend you handle signing/being covered by a CLA _before_ you send a PR to avoid problems, though this is not absolutely necessary until your code is ready to be merged in.
+To agree to the OpenJSF Contributor License Agreement, visit https://cla-assistant.io/ampproject/amphtml/, read through the agreement, and click "Login with GitHub to Agree."
```

Comment:
Turns out we have confusing behavior from the check.

1. For `org/repo` URLs, `https://cla-assistant.io/ampproject/amphtml` (without the trailing slash) works, but `https://cla-assistant.io/ampproject/amphtml/` (with the slash) ends in a 404.
2. For `org`-only URLs, `https://cla-assistant.io/ampproject` (without the trailing slash) ends in a 404, but `https://cla-assistant.io/ampproject/` (with the slash) works.

I'm pretty sure this is a quirk of the CLA Assistant [implementation](https://github.com/cla-assistant/cla-assistant). I'll send out another commit to fix the URLs again.

Meanwhile, can someone confirm that you're seeing this too?

Reply:
This is pretty bad. And I can confirm the behavior you're describing.

Reply:
I've made the URL explicit by hyperlinking it. [`19000e0` (#30793)](https://github.com/ampproject/amphtml/pull/30793/commits/19000e053d7ae078142743ea383a38b4bb6db65b).